docs/requirements.txt

```diff
@@ -15,3 +15,4 @@ snowballstemmer==1.9.0
 sphinx==2.1.2
 sphinxcontrib-httpdomain==1.7.0
 sphinx_rtd_theme==0.4.3
+slackclient==2.8.0
```

Comment:
We should setup this requirements.txt in a different directory.

Reply:
yeah. that is me adding it for mistake and forgot to delete it so far.